scripts/tracing_scripts.py

```diff
@@ -0,0 +1,93 @@
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+import json
+import pickle
+
+
+def generate_key_values():
+    """ Generates key value pairs of public and private keys"""
+    with open("keys.json", "r") as f:
+        keys = json.load(f)
+    f.close()
```

Comment:
```suggestion
```
if you're using `with`, then there is no need to close because it does it for you automatically.